example.env

```diff
@@ -3,6 +3,9 @@ MYSQLUSER=MYSQLUSER
 MYSQLPASS=MYSQLPASS
 MYSQLDB=MYSQLDB
 MYSQLPORT=MYSQLPORT
+GAME_SERVER_HOST=localhost
+GAME_SERVER_PORT=54321
+MOCK_GAME_ONLINE=true
```

Comment:
`MOCK_GAME_SERVER_ONLINE` is what's used in `api/v1/misc.js` but `MOCK_GAME_ONLINE` in the `example.env`.

Reply:
Woops, seems like I hadn't pressed save in my editor, so it wasn't included in the commit. Good catch.